src/components/VmDialog/index.js

```diff
@@ -374,6 +381,7 @@ class VmDialog extends React.Component {
             placeholder='VM Memory'
             value={this.state.memory / 1024 / 1024}
             onChange={this.onChangeVmMemory}
+            fieldHelp={<FieldHelp title='Memory' content='Total memory the virtual machine will be created with. In megabytes.' />}
```

Comment:
I'd unify with the below CPU terminology. Either both should say that "equipped with" or both "created with".